pkg/cmd/util/tokencmd/request_token.go

```diff
@@ -45,17 +45,14 @@ func RequestToken(clientCfg *kclient.Config, reader io.Reader, defaultUsername s
 
 	osClient.Client = &challengingClient{httpClient, reader, defaultUsername, defaultPassword}
 
-	result := osClient.Get().AbsPath("oauth", "authorize").Param("response_type", "token").Param("client_id", "openshift-challenging-client").Do()
+	result := osClient.Get().AbsPath(server.OpenShiftOAuthAPIPrefix, osinserver.AuthorizePath).Param("response_type", "token").Param("client_id", "openshift-challenging-client").Do()
 
 	if len(tokenGetter.accessToken) == 0 {
-		if result.Error() != nil {
-			glog.Errorf("Error making server request: %v", result.Error())
-		}
-
-		requestTokenURL := clientCfg.Host + "/oauth" /* clean up after auth.go dies */ + tokenrequest.RequestTokenEndpoint
-		return "", errors.New("Unable to get token.  Try visiting " + requestTokenURL + " for a new token.")
+		return "", result.Error()
 	}
 
+	clientCfg.BearerToken = tokenGetter.accessToken
```

Comment:
Oh, it is here and new.  Don't mutate the incoming clientCfg.  See comment above, only now I know the ripples can't go too far. :)